frontend/src/components/common/FilePicker.js

```diff
@@ -13,11 +13,13 @@ const propTypes = {
   uploadUrl: PropTypes.string.isRequired,
   existingFilesDropdown: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
   onSelectExisting: PropTypes.func.isRequired,
+  onFileLoad: PropTypes.func,
 };
 
 const defaultProps = {
   maxFileSize: undefined,
   acceptedFileTypesMap: undefined,
+  onFileLoad: undefined,
```

Comment:
defaultProps shouldn't be set to be undefined, set them according to their proptype, if the prop is an array, initial it with an empty array etc. 
```
maxFileSize: " " //set an actual default size here
acceptedFileTypesMap: {},
onFileLoad: () => {},
```

Reply:
The reason they are set to undefined in FilePicker is because I don't want to overwrite or duplicate the defaults that are set in FileUpload.

Reply:
I would choose to not have a defaultProps in FilePicker but its a linting error.